src/layouts/Sidebar.jsx

```diff
@@ -102,8 +104,7 @@ const Sidebar = ({ menuList, activeMenuItem }) => {
         {!isDashboardPage && (
           <div
             className="batch-switch-container flex"
-            onClick={showDrawer}
-            // onClick={() => setShowSwitchBatch(!showSwitchBatch)}
+            onClick={user.role !== "Student" && showDrawer}
           >
             <div className="batch-content-container flex">
```

Comment:
dont show cursor pointer for students when they hover over batch

Reply:
Solved